### PR TITLE
[Testing] non-memmap TIFF

### LINKED ISSUE #100: [Testing]: Add example of non-memmap TIFF file
Ben may have run into one during the Froemke conversion, could add it to the GIN ~~behavior~~ ophys repo for the CI to use.

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
fix #100

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -24,15 +24,15 @@ jobs:
 
       - if: ${{ matrix.os == 'ubuntu-latest' }}
         name: Get datalad - Linux
-        run: conda install -c conda-forge datalad==0.16.1
+        run: conda install -c conda-forge datalad==0.16.3
```
Comment (h-mayorquin) [inline .github/workflows/testing.yml:27]: Should we do the same update on nwb-conversion-tools?

**Review by h-mayorquin — COMMENTED** [review]
This will always through a warning, right?

https://github.com/catalystneuro/roiextractors/blob/40c1ad88bbf8eb8f031254a78bfff57d66b48742/src/roiextractors/extractors/tiffimagingextractor/tiffimagingextractor.py#L48-L54

If we are going to have that warning there should't we have something in the testing that catches it and, for this round trip test, suppresses it?

**Comment by CodyCBakerPhD** [comment]
> If we are going to have that warning there should't we have something in the testing that catches it and, for this round trip test, suppresses it?

Oh, sure good point

**Comment by CodyCBakerPhD** [comment]
> If we are going to have that warning there should't we have something in the testing that catches it and, for this round trip test, suppresses it?

@h-mayorquin Added now

**Review by h-mayorquin — APPROVED** [review]

**Pull request merged**